packages/jest-message-util/src/__tests__/messages.test.ts

```diff
@@ -413,3 +413,20 @@ it('getTopFrame should return a path for mjs files', () => {
 
   expect(frame!.file).toBe(expectedFile);
 });
+
+it('should return the error cause if there is one', () => {
+  const error = new Error('Test exception');
+  // We need to do it this way because in older Javascript engines the constructor does not allow supplying a cause.
+  error.cause = new Error('Cause Error');
+  const message = formatExecError(
+    error,
+    {
+      rootDir: '',
+      testMatch: [],
+    },
+    {
+      noStackTrace: false,
+    },
+  );
+  expect(message).toEqual(expect.stringContaining('Cause'));
```

Comment:
Perhaps it makes sense to snapshot it:

```suggestion
  expect(message).toMatchSnapshot();
```